invokeai/frontend/web/public/locales/en.json

```diff
@@ -143,6 +143,7 @@
         "alpha": "Alpha",
         "selected": "Selected",
         "viewer": "Viewer",
+        "controlLayers": "Control Layers",
```

Comment:
There's already a translation key for this down in the control layers section, but it has ` (BETA)` appended to it. Let's use that, removing ` (BETA)`. We need to be more careful with duplicate strings, there are a crapload and it's more work for both the app and our translators